apps/frontend/src/libs/components/select/styles.module.css

```
@@ -0,0 +1,96 @@
+.container {
+	display: flex;
+	flex-direction: column;
+	gap: 8px;
+}
+
+.title {
+	font-size: 16px;
+	font-weight: 500;
+	color: var(--color-text-primary);
+}
+
+.control {
+	padding: 10px 16px 10px 10px;
+	background-color: var(--color-background-secondary);
+	border: var(--color-border-primary) 1px solid;
+	border-radius: 6px;
+}
+
+.single-value {
+	font-size: 16px;
+	font-weight: 500;
+	color: var(--color-text-primary);
+}
+
+.multi-value {
+	display: flex;
+	gap: 4px;
+	padding-right: 4px;
+	padding-left: 4px;
+	margin-right: 2px;
+	margin-left: 2px;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-hover);
+	border-radius: 4px;
+}
+
+.multi-value-label {
+	font-size: 16px;
+	font-weight: 400;
+	color: var(--color-text-primary);
+}
+
+.indicators-container {
+	color: var(--color-text-secondary);
+}
+
+.menu {
+	position: absolute;
+	top: 100%;
+	z-index: 1;
+	width: 100%;
+	margin-bottom: 8px;
+	background-color: var(--color-background-secondary);
+	border-radius: 4px;
+	box-shadow:
+		0deg 0deg 0deg 1px var(--box-shadow),
+		0deg 4px 11px var(--box-shadow);
+}
+
+.placeholder {
+	font-size: 16px;
+	font-weight: 400;
+	color: var(--color-text-secondary);
+}
+
+.option {
+	width: 100%;
+	padding: 8px 12px;
+	font-size: inherit;
+	color: var(--color-text-primary);
+	border-radius: 4px;
+}
+
+.control--focused {
```

We don't use bem here because of css modules. Just use kebab-case
```suggestion
.control-focused {
```